Src/Witsml/Data/MudLog/WitsmlMudLogLithology.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+using Witsml.Data.Measures;
+
+namespace Witsml.Data.MudLog
+{
+    public class WitsmlMudLogLithology
+    {
+        [XmlAttribute("uid")]
+        public string Uid { get; set; }
+
+        [XmlElement("type")]
+        public string Type { get; set; }
+
+        [XmlElement("codeLith")]
+        public string CodeLith { get; set; }
+
+        [XmlElement("lithPc")]
+        public WitsmlIndex LithPc { get; set; }
+
+        [XmlElement("description")]
+        public string Description { get; set; }
+
+        [XmlElement("lithClass")]
```

Comment:
TThis was also flagged as deprecated, we should include it but along with `dunhamClass` it could be marked in code as deprecated